src/site/stages/build/process-cms-exports/schemas/transformed/paragraph-downloadable_file.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
         entityType: { enum: ['paragraph'] },
         entityBundle: { enum: ['downloadable_file'] },
         fieldMarkup: { type: ['string', 'null'] },
-        fieldMedia: { $ref: 'transformed/media-document' },
+        fieldMedia: { $ref: 'Media' },
```

Comment:
This was throwing an error when I ran `npm run build:content:test`. I'm not sure if this is the right solution, but apparently there was a `media-image` in one of these, so :man_shrugging:

Alternatively, we could specify only the `media-*` schemas we want in a `oneOf`, instead of all of them with `$ref: 'Media'`.

Reply:
Interesting - I'll take a look at this.